packages/host/src/components/AppDrawer/AppDrawer.test.tsx

```diff
@@ -93,7 +93,7 @@ describe('AppDrawer', () => {
       expect(getByText(/suppliers/i)).not.toBeVisible();
       expect(getByText(/stock/i)).not.toBeVisible();
       expect(getByText(/tools/i)).not.toBeVisible();
-      expect(getByText(/reports/i)).not.toBeVisible();
+      // expect(getByText(/reports/i)).not.toBeVisible();
```

Comment:
I removed `reports` from the menu, since we don't have any report functionality from the API. that's a v1+ feature.. though the printing endpoint might give us scope to do something now.. 🤔 

anyway.. simple fix for the test!